hack/testing/test-367-logforwarding.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# Jira LOG-367 - Log forwarding
+
+set -e
+if [ -n "${DEBUG:-}" ]; then
+    set -x
+fi
+
+source "$(dirname $0)/../common"
+
+start_seconds=$(date +%s)
+os::test::junit::declare_suite_start "${BASH_SOURCE[0]}"
+
+ARTIFACT_DIR=${ARTIFACT_DIR:-"$repo_dir/_output"}
+if [ ! -d $ARTIFACT_DIR ] ; then
+  mkdir -p $ARTIFACT_DIR
+fi
+
+cleanup(){
+  os::log::info "Running cleanup"
```

Comment:
I think `local return_code=$?` has to go above the os::log::info call

Else it'll get the value of the call to the info command and return success instead of catching the error.